canonical/snapcraft.io
# Fix selected brandstore not persisting

## Done
- Fix selected brand store not persisting if an error occur when trying to register snap name
## How to QA
- Go to https://snapcraft-io-4506.demos.haus/register-snap
- Select a brand store
- Fill in `snapd` or any registered snap name into the snap name field
- click on register, an error will be displayed 
- Ensure that the store displayed in the store field is the same store you selected earlier
## Issue / Card https://warthogs.atlassian.net/browse/WD-8313
Fixes # 

## Screenshots
